### PR TITLE
China support

### PR DESCRIPTION
**Pull request by jonasman**

### CODE REVIEW

**Comment by jonasman** [comment]
Reopened again with a branch in this repo for extra fixes.
https://github.com/jonasman/TeslaSwift/pull/77

@RSickenberg

**Comment by RSickenberg** [comment]
@jonasman Ok for me but you closed the branch on your side so we cannot merge it for everyone. :-/

File: Sources/TeslaSwift/TeslaSwift.swift [hunk 1]
```diff
@@ -233,7 +234,7 @@ extension TeslaSwift {
                     case .failure(let error):
                         if case let TeslaError.networkError(error: internalError) = error {
                             if internalError.code == 302 || internalError.code == 403 {
-                                self.request(.oAuth2Token, body: body, withExtension: "cn", completion: completion)
+                                self.request(.oAuth2TokenCN, body: body, completion: completion)
```
Comment (birdmichael) [inline Sources/TeslaSwift/TeslaSwift.swift:237]: miss self.token = token

**Review by birdmichael — COMMENTED** [review]
@RSickenberg

**Pull request merged**